fe/fe-core/src/main/java/com/starrocks/sql/common/PCell.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.sql.common;
+
+/**
+ * A cell means the basic unit of the partition, which can be a range partition, a list partition, etc.
+ * eg:
+ * For range partition, it can be {@code Range<PartitionKey>} to represent the range of the partition.
+ * For list partition, it can be {@code List<List<String>>} to represent the list values of the partition.
+ */
+public abstract class PCell {
```

Comment:
It's an empty class for now.

Because there are too many functions only considered `Range<PartitionKey>`  or `List<List<String>>` for now and it's difficult and a big deal to unify all existed usages by `PCell`.

This is only introduced:
1. Simplify the partition values codes.
2. To be used for common inputs later.(eg, use `PCell` to represent `PRangeCell` and `PListCell` both.
We can do this later.